samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/authz/AuthorizationBean.java

```diff
@@ -446,10 +448,10 @@ public boolean isUserAllowedToCreateAssessment() {
 
   // Check whether the assessment belongs to the given site
   public static boolean isAssessmentInSite(final String assessmentId, String siteId, final boolean published) {
-	//Try to get the site Id
```

Comment:
I think we can respect these lines:

if (StringUtils.isBlank(siteId)) {
    siteId = AgentFacade.getCurrentSiteId();
    if (StringUtils.isBlank(siteId)) {
       siteId = SessionManager.getCurrentToolSession().getAttribute("site.instance.id").toString();
    }    
}